pkg/resolver/search.go

```diff
@@ -344,21 +385,43 @@ func formatDataMap(data map[string]interface{}) map[string]interface{} {
 
 func WhereClauseFilter(input *model.SearchInput) []exp.Expression {
 	var whereDs []exp.Expression
-	for _, filter := range input.Filters {
-		if len(filter.Values) > 0 {
-			values := make([]string, len(filter.Values))
-			for i, val := range filter.Values {
-				values[i] = *val
+
+	if input.Keywords != nil {
+		if len(input.Keywords) > 0 {
+			keywords := make([]string, len(input.Keywords))
+			for i, word := range input.Keywords {
+				keywords[i] = "%" + *word + "%"
 			}
+			if len(input.Keywords) == 1 {
+				whereDs = append(whereDs, goqu.L(`"value"`).Like(keywords).Expression())
+			} else if len(input.Keywords) > 1 {
+				for _, key := range keywords {
+					whereDs = append(whereDs, goqu.L(`"value"`).Like(key).Expression())
+				}
 
-			if filter.Property == "cluster" {
-				whereDs = append(whereDs, goqu.C(filter.Property).In(values).Expression())
-			} else {
-				whereDs = append(whereDs, goqu.L(`"data"->>?`, filter.Property).In(values).Expression())
 			}
 		} else {
-			klog.Warningf("Ignoring filter [%s] because it has no values", filter.Property)
+			klog.Warningf("Ignoring filter [%s] because it has no values", input.Keywords)
```

Comment:
Small update to this message.
```suggestion
			klog.Warningf("Ignoring keyword filter [%s] because it has no values", input.Keywords)
```